widget/info.json

```diff
@@ -6,7 +6,7 @@
         "certified": "Yes",
         "publisher": "Fortinet",
         "snapshots": [
```

Comment:
remove **/latest/**
URL Should be -
https://repo.fortisoar.fortinet.com/fsr-widgets/socManagement-1.0.0/images/soc-management.png